plugins/ltac/g_obligations.ml4

```diff
@@ -82,25 +83,37 @@ let next_obligation obl tac = with_tac (fun t -> Obligations.next_obligation obl
 
 let classify_obbl _ = Vernacexpr.(VtStartProof ("Classic",Doesn'tGuaranteeOpacity,[]), VtLater)
 
+
+let pr_obligation_reference _prc _prlc _prt = function
+  | OblNumber i -> Pp.int i
+  | OblName id -> Nameops.pr_id id
```

Comment:
Actually after rebasing and compiling with API, it gave me the deprecation warning which is what I'd expect.